python/source/data.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Data Manipulation
+=================
+
+Recipes related to filtering or transforming data in
+arrays and tables.
+
+.. contents::
+
+See :ref:`compute` for a complete list of all available compute functions
+
+Computing Mean/Min/Max values of an array
+=========================================
+
+Arrow provides compute functions that can be applied to arrays.
+Those compute functions are exposed through the :mod:`arrow.compute`
+module.
+
+.. testsetup::
+
+  import numpy as np
+  import pyarrow as pa
+
+  arr = pa.array(np.arange(100))
+
+Given an array with all numbers from 0 to 100
```

Comment:
We should change this as we have 100 numbers from 0 - 99 (100) while this sentence might lead someone to think these include all the numbers from 0 to 100 (inclusive) i.e. 101 numbers.

Reply:
rephrased